service/storage_service.cc

```diff
@@ -1595,6 +1595,8 @@ future<> topology_coordinator::run() {
             }
         } catch (raft::request_aborted&) {
             slogger.debug("raft topology: topology change coordinator fiber aborted");
+        } catch (raft::commit_status_unknown&) {
```

Comment:
Please explain in the commit message in what situations we currently expect `commit_status_unknown`. Before your PR it should not have happened, but now you introduced the possibility of it happening so we expect it and it's not an error.

Reply:
Done.